easybuild/easyconfigs/s/Siesta/Siesta-4.0-foss-2017b.eb

```diff
@@ -0,0 +1,18 @@
+name = 'Siesta'
+version = '4.0'
```

Comment:
@jrbosch Any reason you're not using the more recent `4.0.1` (for which we already have easyconfig files)?

Reply:
@boegel @migueldiascosta I tried with 4.0.1 with foss toolchain but I got an error during the compilation, I don’t know why this happen, some files don’t exist after that point and easybuild can’t copy some binary file. If you want I can share the log file.

Reply:
tested changed this easyconfig to `4.0.1` (one also needs to change `SOURCELOWER_TGZ` to `SOURCELOWER_TAR_GZ` and update the checksum) and it worked

equivalently, `eb Siesta-4.0.1-intel-2017a.eb --try-toolchain=foss,2017b -dfr`, using the existing `4.0.1` easyconfig, also works

@jrbosch does any of this fail for you?

Reply:
[easybuild@master-slurm ~]$ eb Siesta-4.0.1-intel-2017a.eb --try-toolchain=foss,2017b -dr
== temporary log file in case of crash /tmp/eb-h34hsx/easybuild-BJAyZm.log
== resolving dependencies ...
== processing EasyBuild easyconfig /tmp/eb-h34hsx/tweaked_easyconfigs/Siesta-4.0.1-foss-2017b.eb
== building and installing Siesta/4.0.1-foss-2017b...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== FAILED: Installation ended unsuccessfully (build directory: /opt/easybuild/build/Siesta/4.0.1/foss-2017b): build failed (first 300 chars): Failed to copy file /opt/easybuild/build/Siesta/4.0.1/foss-2017b/siesta-4.0.1/Util/STM/ol-stm/Src/stm to /opt/easybuild/build/Siesta/4.0.1/foss-2017b/siesta-4.0.1/bin: [Errno 2] No such file or directory: '/opt/easybuild/build/Siesta/4.0.1/foss-2017b/siesta-4.0.1/Util/STM/ol-stm/Src/stm'
== Results of the build can be found in the log file(s) /tmp/eb-h34hsx/easybuild-Siesta-4.0.1-20180427.020818.RFAsJ.log
ERROR: Build of /tmp/eb-h34hsx/tweaked_easyconfigs/Siesta-4.0.1-foss-2017b.eb failed (err: "build failed (first 300 chars): Failed to copy file /opt/easybuild/build/Siesta/4.0.1/foss-2017b/siesta-4.0.1/Util/STM/ol-stm/Src/stm to /opt/easybuild/build/Siesta/4.0.1/foss-2017b/siesta-4.0.1/bin: [Errno 2] No such file or directory: '/opt/easybuild/build/Siesta/4.0.1/foss-2017b/siesta-4.0.1/Util/STM/ol-stm/Src/stm'")

[easybuild-Siesta-4.0.1-20180427.020818.RFAsJ.log](https://github.com/easybuilders/easybuild-easyconfigs/files/1954042/easybuild-Siesta-4.0.1-20180427.020818.RFAsJ.log)

Reply:
@jrbosch weird... we have to see what's failing in `ol-stm` but since it's an util, can you meanwhile see if `Siesta` itself works by adding `with_utils = False` to the easyconfig?

Reply:
@migueldiascosta @boegel Yes, Siesta itself works by adding with_utils = False. It seems that the problem is when compiling the ol-stm utility, a syntax error take place at that point.
Another thing I don’t like is “Compilation architecture to be used: x86_64-unknown-linux-gnu—unknown”, due to the unknown term. But it seems to be fine with others util.

Reply:
Any progress on this?